ZupIT/ritchie-cli
# Added changes suggested by goimports

**- What I did**
Hi Ritchiers,

In the search for improving the quality of the code I decided to open a PR by passing the `goimports` in the project.

Adds all suggested changes (and revised one by one) to the project.




**- How to verify it**
> Clone this pr with the command
```bash
git fetch upstream pull/509/head:improvement/updates-linters

```
> And run goimports
```bash
git checkout master
goimports -w $(find . -type f -name '*.go' -not -path "./vendor/*") # Check the differences generated
git checkout improvement/updates-linters
goimports -w $(find . -type f -name '*.go' -not -path "./vendor/*") # Check the differences
```


**- Description for the changelog**

Improves code quality related to linter goimport.
